tests/test_diff.py

```diff
@@ -0,0 +1,64 @@
+# Copyright (c) 2019 The Regents of the University of Michigan
+# All rights reserved.
+# This software is licensed under the BSD 3-Clause License.
+import os
+import unittest
+import signac
+from tempfile import TemporaryDirectory
+
+
+class BaseDiffTest(unittest.TestCase):
+
+    project_class = signac.Project
+
+    def setUp(self):
+        self._tmp_dir = TemporaryDirectory(prefix='signac_')
+        self.addCleanup(self._tmp_dir.cleanup)
+        self._tmp_pr = os.path.join(self._tmp_dir.name, 'pr')
+        self._tmp_wd = os.path.join(self._tmp_dir.name, 'wd')
+        os.mkdir(self._tmp_pr)
+        self.config = signac.common.config.load_config()
+        self.project = self.project_class.init_project(
+            name='testing_test_project',
+            root=self._tmp_pr,
+            workspace=self._tmp_wd)
+        self.project.config['default_host'] = 'testing'
+
+    def tearDown(self):
+        pass
+
+    def open_job(self, *args, **kwargs):
```

Comment:
I don't think we need to define this. Remove this method and use `self.project.open_job` instead of `self.open_job` (you might already be doing this everywhere).